fw.lua

```diff
@@ -325,7 +304,69 @@ local function _log_event(self, request_client, request_uri, rule, match)
 		end
 	}
 
-	lookup[self._event_log_target](t)
+	lookup[self._event_log_target](log_event)
+end
+
+-- constucts the log entry, and pushes it to the configured logging target
+local function _log_request(self, ctx, collections)
+	if (table.getn(ctx.matched_rules) ~= 0) then
+		local t = {
+			timestamp = ngx.time(),
+			client = collections["IP"],
+			method = collections["METHOD"],
+			uri = collections["URI"],
+			alert = ctx.matched_rules,
+			score = ctx.score
+		}
+
+		if self._event_log_request_arguments then
+			t.uri_args = ngx.var.request_uri
```

Comment:
actually, why not have the URI args in the same nested format as the headers? should we not be consistent with how we present that data?